admin/dashboard/assets/src/dashboard-app/pages/fonts/AddFont.js

```diff
@@ -72,7 +63,8 @@ const AddFont = () => {
 					{/* Here will be Font Preview Section */}
 					<div className="pb-3 flex justify-between items-center tablet:block">
 						<div className="text-base font-medium text-secondary">
-							{__('Font Preview', 'custom-fonts')}
+							{ activeType === "local" ? localFont.font_name ? localFont.font_name : __('Font Preview', 'custom-fonts') : null }
```

Comment:
Please use Yoda Formatting for condition checking

Reply:
Updated.